fieldmanager.php

```diff
@@ -3,7 +3,7 @@
  * Fieldmanager Base Plugin File.
  *
  * @package Fieldmanager
- * @version 1.0.0-beta.2
+ * @version 1.0.0-beta.3
```

Comment:
Ideally yes -- I included it here because of the deprecations (even if only in phpdoc).